### ISSUE TITLE
Now an option for iCalendar

### ISSUE THREAD

**Issue by ThetaSinner** [not-yet-awesome-rust/not-yet-awesome-rust#105]
I've been working on a higher level library for iCalendar in Rust -> https://github.com/EphyraSoftware/aetolia

I was wondering if you would consider listing this as an option?

**Comment by ErichDonGubler** [comment]
Absolutely! Feel free to make a PR, cross out the original pain point, and note that you have a (maybe still WIP) solution. 🙂

**Comment by ThomasdenH** [comment]
There is now also [`icalendar_rs`](https://github.com/hoodie/icalendar-rs) which seems to be pretty high level as well, even if documentation could be improved.
